grpc-proto/src/test/java/ValuesTest.java

```diff
@@ -0,0 +1,323 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.stargate.grpc.Values;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.nio.ByteBuffer;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.UUID;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+public class ValuesTest {
+
+  @Nested
+  class BoolTest {
+
+    @Test
+    public void encodeDecode() {
+      boolean expected = true;
+      assertThat(Values.bool(Values.of(expected))).isEqualTo(expected);
+    }
+
+    @Test
+    public void invalid() {
+      assertThatThrownBy(
+              () -> {
+                Values.bool(Values.NULL);
+              })
+          .isInstanceOf(IllegalArgumentException.class);
+    }
+  }
+
+  @Nested
+  class IntTest {
+
+    @Test
+    public void encodeDecode() {
+      int expected = Integer.MAX_VALUE;
+      assertThat(Values.int_(Values.of(expected))).isEqualTo(expected);
+    }
+
+    @Test
+    public void invalid() {
```

Comment:
Would it also be possible to test value overflow, since it is checked by converter?

Reply:
done